src/hmem_cuda.c

```diff
@@ -56,7 +56,7 @@ struct cuda_ops {
 	CUresult (*cuPointerGetAttribute)(void *data,
 					  CUpointer_attribute attribute,
 					  CUdeviceptr ptr);
-	CUresult (*cuPointerSetAttribute)(void *data,
+	CUresult (*cuPointerSetAttribute)(const void *data,
```

Comment:
Leaving comment here, since I can't comment on the commit message directly.  Please use hard returns at the end of lines, about 80 characters, in the commit message.  Otherwise, the text does not show properly when viewing via git log.

Reply:
done